aws/monolith-terraform/xebialabs/xld-service.yaml.tmpl

```diff
@@ -0,0 +1,34 @@
+{{$app := .AppName | kebabcase}}
+{{$region := "{{AWSRegion}}"}}
+{{$aws_access_key := "{{AWSAccessKey}}"}}
+{{$aws_secret_key := "{{AWSSecretKey}}"}}
+{{$mysql_address := "{{MYSQL_DB_ADDRESS}}"}}
+{{$mysql_port := "{{MYSQL_DB_PORT}}"}}
+apiVersion: xl-deploy/v1
+kind: Applications
+spec:
+- name: {{$app}}
+  type: core.Directory
+  children:
+    - name: {{$app}}-terraform-service
+      type: udm.Application
+      lastVersion: '1.0'
+      children:
+      - name: '1.0'
+        type: udm.DeploymentPackage
+        deployables:
+        - name: {{$app}}-terraform-service
+          type: terraform.Module
+          file: !file ../terraform/service/service.zip
+          targets:
+            - 'aws_ecs_task_definition.ecs-task-definition'
+            - 'aws_ecs_service.ecs-service'
+          inputVariables:
```

Comment:
I think this needs output declaration to get ALB_NAME

Reply:
outputVariables are defined in the output.tf file. The load balancer is defined in the infrastructure module.